homeassistant/components/assist_satellite/entity.py

```diff
@@ -0,0 +1,151 @@
+"""Assist satellite entity."""
+
+from collections.abc import AsyncIterable
+import time
+from typing import Final
+
+from homeassistant.components import stt
+from homeassistant.components.assist_pipeline import (
+    OPTION_PREFERRED,
+    AudioSettings,
+    PipelineEvent,
+    PipelineEventType,
+    PipelineStage,
+    async_get_pipelines,
+    async_pipeline_from_audio_stream,
+    vad,
+)
+from homeassistant.const import EntityCategory
+from homeassistant.core import Context
+from homeassistant.helpers import entity
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.util import ulid
+
+from .models import AssistSatelliteState
+
+_CONVERSATION_TIMEOUT_SEC: Final = 5 * 60  # 5 minutes
+
+
+class AssistSatelliteEntity(entity.Entity):
+    """Entity encapsulating the state and functionality of an Assist satellite."""
+
+    entity_description = EntityDescription(
+        key="assist_satellite",
+        translation_key="assist_satellite",
+        entity_category=EntityCategory.CONFIG,
+    )
```

Comment:
Why do we use entity descriptions by default? It serves no extra purpose and can be replaced by `_attr_translation_key` and  `_attr_entity_category`

Reply:
When removing `entity_description` I get `AttributeError` everywhere with "object has no attribute 'entity_description'"